tests/test_cli_compile.py

```diff
@@ -2582,6 +2582,37 @@ def test_input_formats(fake_dists, runner, make_module, fname, content):
     assert "extra ==" not in out.stderr
 
 
+@pytest.mark.network
```

Comment:
```suggestion
```

This test should not require the network, because using `--no-build-isolation`.

Reply:
Thanks, implemented!